### PR TITLE
Add more information on how to set up and use Remixer

### LINKED ISSUE #52: [README] All-in-one dependency is not working
`compile 'com.github.material-foundation:material-remixer-android:develop-SNAPSHOT'` causes the error message 
```
Error:Module 'com.github.material-foundation:material-remixer-android:develop-SNAPSHOT' depends on one or more Android Libraries but is a jar
```

Short term fix is to depend on each module separately:
```
compile 'com.github.material-foundation.material-remixer-android:remixer_core:develop-SNAPSHOT'
compile 'com.github.material-foundation.material-remixer-android:remixer_ui:develop-SNAPSHOT'
compile 'com.github.material-foundation.material-remixer-android:remixer_storage:develop-SNAPSHOT'
 ```

Long term fix TBD. We will ideally have a all-in-one target that clients can depend on for ease of use.

### PR DESCRIPTION
**Pull request by miguelandres**
- Expand the dependency information since the single-dependency version does not actually work
- Explain what one-time set up is required on the Application class.
- Explain how to set up the Activity.onCreate and what each of those lines means.
- Give examples of how to use the annotations.

Fixes #52 #53 #59

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -55,13 +57,114 @@ And in your modules, apply the `android-apt` plugin and add the remixer dependen
...
+### How to use it in an activity
+
+__Only in the activities where you're using remixer__
+
+You need to add a few lines at the end of your `Activity.onCreate()`
+
+1. `RemixerBinder.bind(this);` creates, initializes and sets up all the Variables and trigger you define in this activity.
+2. `RemixerFragment remixerFragment = RemixerFragment.newInstance();` Creates the fragment that will be shown when remixer is invoked, then you need at least one of the following:
+  - A variation of `remixerFragment.attachToGesture(this, Direction.UP, 3);`, this example ties showing the Remixer Fragment on a 3-finger swipe up.
+  - `remixerFragment.attachToButton(this, someButtonObject);`, this makes the OnClickListener for a button open the Remixer fragment
```
Comment (chuga) [inline README.md]: nit: missing a dot at the end of the sentence

File: README.md [hunk 2]
```diff
@@ -55,13 +57,114 @@ And in your modules, apply the `android-apt` plugin and add the remixer dependen
...
+```
+
+### How to use it in an activity
+
+__Only in the activities where you're using remixer__
+
+You need to add a few lines at the end of your `Activity.onCreate()`
+
+1. `RemixerBinder.bind(this);` creates, initializes and sets up all the Variables and trigger you define in this activity.
+2. `RemixerFragment remixerFragment = RemixerFragment.newInstance();` Creates the fragment that will be shown when remixer is invoked, then you need at least one of the following:
```
Comment (chuga) [inline README.md]: To be consistent, this sentence should start in lowercase like the others

File: README.md [hunk 3]
```diff
@@ -55,13 +57,114 @@ And in your modules, apply the `android-apt` plugin and add the remixer dependen
...
+public void setTitleColor(Integer color) {
+}
+```
+
+A trigger to simulate an event happening:
+```java
+@TriggerMethod
+public void simulateConnectionFailure() {
+}
+```
```
Comment (chuga) [inline README.md:164]: nit: should one of the examples also set the key of the variable?

**Review by chuga — APPROVED** [review]
Just a few minor comments

File: README.md (hunk 3 above)
Comment (miguelandres) [reply README.md:164]: Done.

File: README.md (hunk 1 above)
Comment (miguelandres) [reply README.md]: Done

File: README.md (hunk 2 above)
Comment (miguelandres) [reply README.md]: Done.

**Pull request merged**